waku/protocol/v2/waku_store.nim

```diff
@@ -55,7 +56,7 @@ proc encode*(response: HistoryResponse): ProtoBuffer =
   for msg in response.messages:
     result.write(2, msg.encode())
 
-proc query(w: WakuStore, query: HistoryQuery): HistoryResponse =
+proc findMessages(w: WakuStore, query: HistoryQuery): HistoryResponse =
```

Comment:
public function?

Reply:
No, this is only needed by the protocol.